main.go

```diff
@@ -196,7 +199,8 @@ If no version is passed, the pulumi/{pkg,sdk} version will track the bridge`)
 - "provider": Upgrade the upstream provider only.
 - "sdk": Upgrade the Pulumi sdk only.
 - "pf": Upgrade the Plugin Framework only.
-- "code":     Perform some number of code migrations.`)
+- "code":     Perform some number of code migrations.
+- "check-upstream-version": Determine if we need to upgrade the upstream provider. For use in CI only."`)
```

Comment:
"For use in CI only" this seems an odd restriction to make? Why is that?

Reply:
The issue check will most likely fail because the pr author we check for is `@me` - so, for CI, pulumi-bot. If you run locally, then the behavior is differnet.

I'm also happy to remove any documentation here altogether and keep this info in our playbooks instead.